src/herder/test/TransactionQueueTests.cpp

```diff
@@ -207,11 +209,12 @@ class TransactionQueueTest
 };
 }
 
-TEST_CASE("TransactionQueue", "[herder][TransactionQueue]")
+TEST_CASE("TransactionQueue", "[herder][transactionqueue]")
 {
     VirtualClock clock;
     auto cfg = getTestConfig();
     cfg.TESTING_UPGRADE_MAX_TX_SET_SIZE = 4;
+    cfg.FLOOD_TX_PERIOD_MS = 100;
```

Comment:
I've enabled this in tests that actually check that we flood with this new code.
When we make this the default, I will probably do a sweep to enable it in more cases (probably will flip things around).

That said, many tests depend on timing, so trying to run them with different flooding strategies will not work without actually changing them (which in some cases we should fix), but in many cases it's actually not useful anyways as the only thing we care about is that flooding happens, the tests are testing things unrelated to flooding. Example: we have tests that submit a tx, then close a ledger. Now we'd have to "time things right" so that the transaction gets included in that first ledger.